packages/components/pages/f-account-info/test/component/f-account-info.component.spec.js

```diff
@@ -0,0 +1,96 @@
+import forEach from 'mocha-each';
+
+const AccountInfo = require('../../test-utils/component-objects/f-account-info.component');
+
+let accountInfo;
+
+describe('f-account-info component tests', () => {
+    beforeEach(() => {
+        accountInfo = new AccountInfo();
+
+        accountInfo.load();
+
+        accountInfo.waitForComponent();
+    });
+
+    it('should display the f-account-info component', () => {
+        // Assert
+        expect(accountInfo.isComponentDisplayed()).toBe(true);
+    });
+
+    it('should show that the email formfield is disabled', () => {
+        // Assert
+        expect(accountInfo.isDisabled('emailAddress')).toBe(true);
+    });
+
+    it('should check if all call to actions are clickable', () => {
```

Comment:
Might be worth changing this to a generic `canBeClicked(buttonName)` function and using `mocha-each` to for `forEach` this test.

Currently, I feel if an assert fails, it won't execute future asserts. This can be avoided with the above approach.